tests/test_surfaces.py

```diff
@@ -116,3 +128,31 @@ def test_misc(self):
             c.R_lmn = s.R_lmn
         with pytest.raises(ValueError):
             c.Z_lmn = s.Z_lmn
+
+        assert c.sym
+
+
+def test_rotation_matrix():
+    A = rotation_matrix([0, 0, np.pi / 2])
+    At = np.array([[0, -1, 0], [1, 0, 0], [0, 0, 1]])
+    np.testing.assert_allclose(A, At, atol=1e-10)
+
+
+def test_xyz2rpz():
+    xyz = np.array([1, 1, 1])
+    rpz = xyz2rpz(xyz)
+    np.testing.assert_allclose(rpz, [np.sqrt(2), np.pi / 4, 1], atol=1e-10)
+
+    xyz = np.array([0, 1, 1])
+    rpz = xyz2rpz_vec(xyz, x=0, y=1)
+    np.testing.assert_allclose(rpz, np.array([[1, 0, 1]]), atol=1e-10)
+
+
+def test_rpz2xyz():
+    rpz = np.array([np.sqrt(2), np.pi / 4, 1])
+    xyz = rpz2xyz(rpz)
+    np.testing.assert_allclose(xyz, [1, 1, 1], atol=1e-10)
+
+    rpz = np.array([[1, 0, 1]])
+    xyz = rpz2xyz_vec(rpz, x=0, y=1)
+    np.testing.assert_allclose(xyz, np.array([[0, 1, 1]]), atol=1e-10)
```

Comment:
Do these tests of functions from `geometry/utils` belong in the `test_surfaces` file? Perhaps it would be better to create a new `test_geometry` file?